nbsite/gallery/README.md

```diff
@@ -0,0 +1,14 @@
+# nbsite-gallery
+
+The nbsite gallery is a Sphinx extension to build a gallery from
+directory of notebook examples containing HoloViews plots.
```

Comment:
It only works with HoloViews plots, not Bokeh plots in general?  If it worked with Bokeh then presumably it would work for Panel, with its other plot types...

Reply:
It'll export panel and bokeh plots just fine but the thumbnailer doesn't handle it yet.